natterstefan/vscode-toggl-extension
# feat: text for new entry can be selected in editor

# Issue

Issue #14  

## What I did

I used selection from active editor to pre-fill the value for command `startEntry`.
I also added better error handling when the user cancels the command `StartExistingEntry`  

## What it adds, solves and/or improves

User can use selected text as a value for new entry.

## How to test

Select the text in the editor and invoke action `Toggl: Start with new entry
Selected text should appear in the InputBox
